src/library/scala/collection/immutable/Vector.scala

```diff
@@ -466,24 +466,24 @@ override def companion: GenericCompanion[Vector] = Vector
       display5 = copyRange(display5, oldLeft, newLeft)
   }
 
-  private def zeroLeft(array: Array[AnyRef], index: Int): Unit = {
+  private[this] def zeroLeft(array: Array[AnyRef], index: Int): Unit = {
```

Comment:
The bytecode for a `private[this]` method is exactly the same as for `private`, so this should not make any difference.